package.json

```diff
@@ -6,10 +6,50 @@
     "node": "10.16.0"
   },
   "dependencies": {
-    "@babel/core": "7.5.5",
     "@material-ui/core": "^4.2.1",
     "@material-ui/icons": "^4.2.1",
-    "@svgr/webpack": "4.3.2",
+    "dotenv": "6.2.0",
+    "dotenv-expand": "4.2.0",
+    "notistack": "^0.4.3",
+    "react": "^16.8.0",
+    "react-app-polyfill": "^1.0.2",
+    "react-dom": "^16.8.0",
+    "react-redux": "^7.1.3",
+    "react-router-dom": "^5.1.2",
```

Comment:
Can we at least consider using Reach Router? I have tried it recently and like it a lot better than React Router tbh.

Reply:
No. https://reacttraining.com/blog/reach-react-router-future/
Also, this one has hooks :D

Reply:
That will be a future refactor tho :v

Reply:
Nice!